<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateOPReturnDialog</class>
 <widget class="QDialog" name="CreateOPReturnDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create OP_RETURN Graffiti</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="placeholderText">
      <string>Enter plain text to be encoded in an OP_RETURN output</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>50</number>
     </property>
     <item>
      <widget class="BitcoinAmountField" name="feeAmount"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonCreate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BitcoinAmountField</class>
   <extends>QLineEdit</extends>
   <header>qt/drivechainamountfield.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
